walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -223,6 +340,35 @@ public void justEnoughRoom() {
     Assert.assertEquals(expected, actual);
   }
 
+  @Test
+  public void notEnoughRoomForOptionalAttendee() {
```

Comment:
Is this exercising a different code path than the test on line 127? They look very similar

Reply:
True, they are very similar! I'll keep it anyways because it's one of the scenarios asked in the guide.